### PR TITLE
Setup project files

### PR DESCRIPTION
**Pull request by montsecruz9**
In this branch I did:
- Add HTML file
- Add CSS file
- Edit README file 
- Add linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (ademibowale) [inline .gitignore:1]: - [ ] I would kindly suggest you to Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).A test.md file with the text "This file should be ignored by git".kindly include text.md in your .gitignore file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    ><link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <h1 class="hello"> Hello Microverse! </h1>
+
```
Comment (ademibowale) [inline index.html:12]: - [ ] I would kindly suggest you to use a semantic html tag according to the project requirement html file that includes a header with the text "Hello Microverse!". Use h1 tag for that pleasefix this its good for better accessibilty.

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @montsecruz9  , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by montsecruz9** [comment]
wen I run command **npx hint .** my console says there are 0 errors

**Comment by montsecruz9** [comment]
image

**Pull request closed without merging**